glean-core/src/metrics/mod.rs

```diff
@@ -77,6 +77,9 @@ pub struct DistributionData {
 
     /// The accumulated sum of all the samples in the distribution.
     pub sum: i64,
+
+    /// The total number of entries in the distribution.
+    pub count: u64,
```

Comment:
Trying to set this as an `i64` caused errors in the Kotlin tests. I am not exactly sure why that might be, but I would prefer this work the same as `sum`.